### PR TITLE
Deprecate confirm types

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-deprecate-confirm-types](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-deprecate-confirm-types)

File: src/app/public/modules/confirm/confirm-type.ts [hunk 1]
```diff
@@ -9,10 +9,12 @@ export enum SkyConfirmType {
   OK = 1,
   /**
    * Displays two buttons with **Yes** and **Cancel** labels.
+   * @deprecated Please use the `Custom` type.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/confirm/confirm-type.ts:12]:
```suggestion
   * @deprecated Use the `Custom` type to follow the guidance that labels should clearly indicate the actions that occur when users select buttons.
```

File: src/app/public/modules/confirm/confirm-type.ts [hunk 2]
```diff
@@ -9,10 +9,12 @@ export enum SkyConfirmType {
...
    */
   YesCancel = 2,
   /**
    * Displays three buttons with **Yes**, **No**, and **Cancel** labels.
+   * @deprecated Please use the `Custom` type.
```
Comment (blackbaud-johnly) [inline src/app/public/modules/confirm/confirm-type.ts:17]:
```suggestion
   * @deprecated Use the `Custom` type to follow the guidance that labels should clearly indicate the actions that occur when users select buttons.
```

**Review by blackbaud-johnly — DISMISSED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-deprecate-confirm-types](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-deprecate-confirm-types)

**Pull request merged**